src/GdbServer.cc

```diff
@@ -1771,7 +1774,14 @@ void GdbServer::launch_gdb(ScopedFd& params_pipe_fd,
   if (!did_set_remote) {
     push_target_remote_cmd(args, string(params.host), params.port);
   }
-  args.push_back(params.exe_image);
+  if (!target_wine) {
```

Comment:
Flip the true/false block order to avoid ! here.

Reply:
Will do.